<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Validation Rules</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Layer #1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="layer1Combo"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Necessity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="necessityCombo">
         <item>
          <property name="text">
           <string>must (be)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>must not (be)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Spatial predicate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="predicateCombo">
         <item>
          <property name="text">
           <string>equal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>disjoint</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>intersect</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>touch</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cross</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>within</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>overlap</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>contain</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cover</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>covered by</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Layer #2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="layer2Combo"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Cardinality</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cardinalityEdit">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>317</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="insertRuleButton">
     <property name="text">
      <string>Insert Rule</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="removeRuleButton">
     <property name="text">
      <string>Remove Rule</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Layer #1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Necessity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Predicate</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Layer #2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cardinality</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
